action.yml

```
@@ -9,6 +9,10 @@ inputs:
     description: 'flags/arguments for command'
     required: false
     default: ''
+  version:
```

Other tools also use prefixes (`go-version`, `java-version`)

```suggestion
  piper-version:
```